Library/Formula/xcconf.rb

```diff
@@ -0,0 +1,13 @@
+class Xcconf < Formula
+  homepage "https://github.com/AlexDenisov/xcconf"
+  url "https://github.com/AlexDenisov/xcconf/archive/0.1.1.tar.gz"
+  sha1 "d3768af9ab739c49ac107393ac19047a962e703a"
+
+  def install
+    system "make", "build"
+    bin.install "./build/Release/xcconf"
```

Comment:
Well, we can, then I have to put Homebrew env variables into my makefile, which I don't really want to do.
Are there any difference/problems with `bin.install`? What's the purpose of this command if you suggest to avoid it?